IMathFigure.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace objected_oriented_programming
+{
+    interface IMathFigure
+    {
+        public double Square();
+    }
```

Comment:
Модификатор доступа не туда поставили.
```suggestion
    internal interface IMathFigure
    {
        double Square();
    }
```